h/static/styles/partials-v2/_annotation-card.scss

```diff
@@ -0,0 +1,60 @@
+// Annotation card
+// ---------------
+// Styles for the annotation card which is displayed in the
+// results of activity queries.
+
+.annotation-card {
+  @include font-normal;
+  border: 1px solid $grey-2;
+  list-style: none;
+  padding: 15px;
+  margin-bottom: 10px;
+  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.1);
+  background: white;
+}
+
+.annotation-card:hover {
+  box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.15);
+
+  .annotation-card__quote {
+    border-left: $highlight 3px solid;
+    color: $grey-5;
+  }
+
+  .annotation-card__timestamp {
+    color: $grey-5;;
+  }
+}
+
+.annotation-card__header {
+  margin-bottom: 15px;
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.annotation-card__username {
+  color: #202020;
+  font-weight: bold;
+}
+
+.annotation-card__timestamp {
+  color: $grey-4;
+  font-size: $small-font-size;
```

Comment:
You can use the `font-small` mixin here to set the font-size and line-height together